docs/source/conf.py

```diff
@@ -16,6 +16,16 @@
 
 ROOT = os.path.abspath(os.path.join("..", ".."))
 sys.path.insert(0, ROOT)
+
+import shutil  # noqa
```

Comment:
Very minor final comment: this should be moved on top of the file and have the `#  noqa` removed. We need the `# noqa` for skcosmo since it needs to come after the `sys.path` manipulation, but there is no reason to have it for shutils